static/src/javascripts/projects/common/modules/identity/upsell/consent-card/FollowCard.js

```diff
@@ -14,31 +14,26 @@ export type CardLike = {
 export type Followable<T: CardLike> = {
     value: T,
     onChange: boolean => void,
+    isFollowing: boolean,
 };
 
-type FollowCardProps<T: CardLike> = {
-    followable: Followable<T>,
-    hasFollowed: boolean,
-};
-
-class FollowCard<T: CardLike> extends Component<FollowCardProps<T>, {}> {
+class FollowCard<T: CardLike> extends Component<Followable<T>, {}> {
```

Comment:
Is it possible to make this a [stateless functional component](https://reactjs.org/docs/components-and-props.html#functional-and-class-components)?